client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,27 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          codeLines[actualLineNumber].classList.add('tie-syntax-error-line');
+        };
+
+        /**
+          * Clears highlight from syntax errors in the coding UI
+          */
+        var clearHighlight = function() {
```

Comment:
Just realized that clearAllHighlights() is probably a better name, since that's what this is doing (as opposed to clearing a single highlight).